src/backend/aspen/api/views/samples.py

```diff
@@ -211,3 +224,60 @@ async def delete_sample(
     await db.delete(sample)
     await db.commit()
     return SampleDeleteResponse(id=sample_db_id)
+
+
+@router.put("/")
+async def update_samples(
+    update_samples_request: UpdateSamplesRequest,
+    db: AsyncSession = Depends(get_db),
+    settings: Settings = Depends(get_settings),
+    user: User = Depends(get_auth_user),
+) -> SamplesResponseSchema:
+
+    # reorganize request data to make it easier to update
```

Comment:
Looking at it some more, I guess it's because in the original creation request, the pathogen_genome and the sample are each sub-objects that make up different aspects? Seems like it could have been bundled up under the sampleId, but it does make more sense in that context.

Not sure if it's worth changing the request format between creation and update, but it already changes somewhat since update doesn't have sub-dicts for pathogen_genome and sample. If you'd rather just have the incoming data fit what you're reorganizing it into, it's certainly no harder for the FE.

Reply:
I really want the API **responses** to send back lists as much as possible because eventually we'll need to add some sort of searching/pagination on the backend and we'll be returning *ordered* results.

As for sub-dicts... since we already require those structures for upload, we should probably keep them similar! I'll update the request format.

Reply:
Thanks for looking into it. Seems like the most reasonable option then.